maven-xml/src/test/java/org/apache/maven/xml/sax/filter/ConsumerPomXMLFilterTest.java

```diff
@@ -231,5 +234,22 @@ public void licenseHeader() throws Exception {
         String actual = transform( input );
         assertThat( actual ).and( expected ).areIdentical();
     }
+    
+    @Test
+//    @Ignore
+    public void lexicalHandler() throws Exception
```

Comment:
Should this test method not describe what it verifies, i.e. "removes modules" or something like that?

Reply:
It tests that the lexical handler is used in both SaxSource and StreamResult. If one of both doesn't provide the lexical handler, all comments will be ignored. By default the SaxSource has no LexicalHandler, so the XMLParser will simply get elements, attributes, etc.

But yes, this is now testing 2 different things. I will add a test with a minimal pom + license, that should be enough to verify if the lexicalHander is properly chained.